### ISSUE TITLE
possible small bug in configurePython

### ISSUE THREAD

**Issue by ablack3** [OHDSI/PatientLevelPrediction#420]
https://github.com/OHDSI/PatientLevelPrediction/blob/7a55c31de7e9a49290c3c35e172962440c70b8a8/R/HelperFunctions.R#L104

Do you want to use asignment here instead of `==`? 
I don't think this code does anything as it is written.

```
configurePython <- function(envname='PLP', envtype=NULL, condaPythonVersion="3.11"){
  
  if(is.null(envtype)){
    if(getOs()=='windows'){
      envtype=='conda'
    } else {
      envtype=='python'
    }
  }
```

changed to 

```
configurePython <- function(envname='PLP', envtype=NULL, condaPythonVersion="3.11"){
  
  if(is.null(envtype)){
    if(getOs()=='windows'){
      envtype <- 'conda'
    } else {
      envtype <- 'python'
    }
  }
```

**Comment by egillax** [comment]
Thanks @ablack3 for catching that. I'll fix it asap.

**Comment by egillax** [comment]
Fixed in v6.3.7
